WordPress/Classes/ViewRelated/Media/Library/MediaViewController.swift

```diff
@@ -102,6 +175,12 @@ final class MediaViewController: UIViewController, NSFetchedResultsControllerDel
             }
         }
         pendingChanges = []
+
+        if let workItem = pendingRefreshWorkItem {
+            workItem.cancel()
+            didFinishRefreshing(error: nil)
+            pendingRefreshWorkItem = nil
```

Comment:
I think you can omit this since didFinishRefreshing(error:) sets pendingRefreshWorkItem to nil